data_structures/indexed_priority_queue.cpp

```diff
@@ -0,0 +1,258 @@
+/**
+ * @file
+ * @brief This is an implementation of an min indexed priority queue,

```

Comment:
Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂

Reply:
just added a link to a Wikipedia page!